### ISSUE TITLE
Suspend a link or node doesn't show status change

### ISSUE THREAD

**Issue by grossmj** [GNS3/gns3-web-ui#1234]
Suspending a link or a node in the web-ui should show the interface labels yellow and the node status icon yellow in the topology summary.

![Screenshot from 2022-01-17 19-41-24](https://user-images.githubusercontent.com/4752614/149740814-4de1a6e4-38da-4aab-acf9-f492d0165f0a.png)

![Screenshot from 2022-01-17 19-42-30](https://user-images.githubusercontent.com/4752614/149741156-0621bd6b-2231-49ce-a35f-8b3f40800d66.png)

I haven't tested with 2.2 but it is likely that issue should be fixed on the web-ui master branch.

**Comment by grossmj** [comment]
I confirm the bug is in version 2.2 as well.

**Comment by grossmj** [comment]
Let's fix this for 3.0 first and we will backport later.

@rajnikantsolarwinds @karunyasolarwinds @sakshisolarwinds 

Notifications such a link status changes are received from the server on a web socket. The address is referenced on  https://github.com/GNS3/gns3-web-ui/blob/master-3.0/src/app/services/notification.service.ts#L22

The event is in JSON format and would look like this:

```
{'action': 'link.updated', 'event': {'capture_compute_id': None, 'capture_file_name': None, 'capture_file_path': None, 'capturing': False, 'filters': {}, 'link_id': '0b186d55-e3f0-4f6f-ae60-f2c261d992c3', 'link_style': {}, 'link_type': 'ethernet', 'nodes': [{'adapter_number': 0, 'label': {'rotation': 0, 'style': 'font-size: 10; font-style: Verdana', 'text': 'e0', 'x': 66, 'y': -9}, 'node_id': 'abfb0e2e-89d7-4636-bbc4-94e9960c95c5', 'port_number': 0}, {'adapter_number': 0, 'label': {'rotation': 0, 'style': 'font-size: 10; font-style: Verdana', 'text': 'e0', 'x': -12, 'y': 57}, 'node_id': '4fd904f4-d64a-494c-aca7-456546b286d4', 'port_number': 0}], 'project_id': '7a696bb1-5c5c-4e7a-bca8-31eed591732a', 'suspend': True}}
```

The event seems to be caught there: https://github.com/GNS3/gns3-web-ui/blob/16d86a23222b3884eb82d567e168f1295c0e0a3f/src/app/handlers/project-web-service-handler.ts#L47L50

If `suspend` is True then the link has been suspended and interface labels should turn yellow.
